code/processes/dqe.q

```diff
@@ -1,7 +1,13 @@
 \d .dqe
 
 configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqeconfig.csv"]];
+dqedbdir:@[value;`dqedbdir;`:dqedb];
 detailcsv:@[value;`.dqe.detailcsv;first .proc.getconfigfile["dqedetailtab.csv"]];
+gmttime:@[value;`gmttime;1b];
+partitiontype:@[value;`partitiontype;`date];
+getpartition:@[value;`getpartition;
+	{{@[value;`.dqe.currentpartition;
```

Comment:
Should this be a nested function which will return a function when called? Looks like you want this to return a date.